safe/metadata/property/character_string_property.py

```diff
@@ -35,8 +33,8 @@ def __init__(self, name, value, xml_path):
         super(CharacterStringProperty, self).__init__(
             name, value, xml_path, self._allowed_python_types)
 
-    @staticmethod
-    def is_valid(value):
+    @classmethod
+    def is_valid(cls, value):
         # any string sequence is valid.
         return True
 
```

Comment:
Please review this changes, is this correct?
The old one raises pylint because it had different args with the base class